CHANGELOG.unreleased.md

```diff
@@ -26,4 +26,7 @@
 > Keep this list alphabetically ordered to avoid merge conflicts
 
 <!--packages-start-->
+
+- xo-server patch
```

Comment:
Also, sorry I forgot to ask earlier, but we should add a bugfix entry for this fix, as it has been reported by users.